### ISSUE TITLE
Monero Community Workgroup Meeting: Saturday 3rd December 2022 @ 16:00 UTC 

### ISSUE THREAD

**Issue by plowsof** [monero-project/meta#759]
# Statement regarding SoloptXMR 

During a [community meeting](https://github.com/monero-project/meta/issues/723#issuecomment-1200262123) 3 months ago, it was decided to withold mj's partial payment of milestone 3 (MS3) in the best interests of the community. This would:
- Prevent project abandonment
- Guarantee that endor had encouragement/help with the uncompleted MS2 + their part of MS3

Fast-forward to today: due to external/unforeseen factors, endor has not yet been able to complete MS2, MS3 or MS4.
We have learned that the working relationship between mj/endor has fallen apart.
mj however, has not only completed their part of MS3 but also MS4. [Endor states:](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/299#note_19778)

> I confirm that I can complete the remainder of my work on M2+M3+M4 independently 
(though of course any collaboration is always welcome), and that mj can be paid for his part of M3+M4 
(20.10 XMR + 20.10 XMR, as per the proposal) right away.
>
> -- <cite>endor</cite>

There has been a public controversy surrounding this proposal. 
The CCS itself is devoid of opinion/emotion regarding 'extra curricular' activities. Work done as per your proposal is work paid. Currently, SoloptXMR:
- Contains no dev tax.
- With initial setup effort from the end user, external readings via sensors/cameras will control a mining rig. 

Now that mj has completed their part of MS3+4 / endor requires no further help / their working relationship is over - 
reasons for withholding MS3 (and now MS4) may need to be reevaluated.

# Meeting info

Location: [Libera.chat, #monero-community](https://libera.chat/) | [Matrix](https://matrix.to/#/#monero-community:monero.social?via=matrix.org&via=monero.social)

[Instructions for joining the monero.social Matrix server.](https://forum.monero.space/d/79-how-to-join-the-monero-core-team-matrix-server-web)

Time
16:00 UTC [Check your timezone](https://www.timeanddate.com/countdown/christmas?iso=20221203T16&p0=1440&msg=Monero+Community+Workgroup+Meeting%3A+Saturday+3rd+December+2022+%40+16%3A00+UTC+&font=sanserif&csz=1)

Moderator: plowsof

Please reach out in advance of the meeting if you would like to propose an agenda item.

Proposed Meeting Items:

1. Introduction
2. Greetings
3. Community highlights    
News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/) - [The Monero Standard](https://localmonero.co/the-monero-standard)
4. [CCS updates](https://ccs.getmonero.org/)    
  a. [Monero Paper Wallets](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/352) - Panda wishes to close while they work on letting the community get to know their work / contributions.
  b. [Metronero checkout](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/353)    
  c. [xmr-btc-swap development and improvement](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/355)    
  d. [CypherPunk Radio](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/357)    
  e. [Bulletproofs++ Peer Review](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/358)    
  f. [MoneroShopping](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/360)    
  g. [rayatina-archive](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/361) 
  h. [Draft: MoneroKon 2023 CCS-1](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/362)  
  i. [escapethe3RA Monero Observer maintenance (Winter 2022)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/363)      
5. Workgroup reports    
  a. Dev workgroup
  b. Localization workgroup
  c. Outreach workgroup
  d. Events workgroup - [MoneroKon 2023](https://github.com/MoneroKon/)
  e. Website workgroup
  f. Policy workgroup
  g. Research workgroup
  h. [Seraphis Migration workgroup](https://github.com/seraphis-migration)
6. Open ideas time    
  - [IRS bounty - Full story by HYC](https://www.reddit.com/r/Monero/comments/z9j62d/the_irs_bounty_the_full_story/)
  - [locked_transfer - Full story by Mochi101](https://www.reddit.com/r/Monero/comments/za1w0r/the_full_locked_transfer_story/)
8. Confirm next meeting date/time    

previous meetings logs #754     

Meeting logs will be posted here afterwards.

**Comment by nahuhh** [comment]
PSA. 
- The 1hr meeting tomorrow only has allows approximately 7 minutes per proposal.
- Monerokon is on the table, im assuming at least 15 minutes.

I'm sick and tired of wasting time listening to this guys demands.
We have better things to do than cater to a known scammer.

Completed milestones = pay.
He wrote the proposal himself, and in his own words _guaranteed_ he would complete endors parts. Around the time he was threatening people, he said he can wait for pay and always waits for the community. This time he's just being a slimeball in public.


Tell him hurry up with that lawsuit.


Example: monero checkout 

Milestone 1 (5 weeks - 50 XMR)

Planning the backend, APIs (admin, merchant account management), multi-language support.

Rewriting the core backend features. Merchant accounts and management of merchant provided checkout page templates.

Rewriting the frontend in Bulma. Current frontend consists of a merchant panel, default checkout page template, authentication and error pages.

Administration and deployment of publicly accesible instance. For the purposes of development and feedback from the community.

**if this was run mj, hed be like "gimme 50% of the money because I did 50% of the milestone**

No. You wrote the milestone the way you wrote it becauase you also guaranteed you would complete it in full.
Snakeoil salesmen

**Comment by mj-xmr** [comment]
> We have learned that the working relationship between mj/endor has fallen apart.

I wouldn't call it this way straight away. I was deeply shocked by endor's reaction and had my own knee-jerk reaction to that, because I had thought that the deal with M4 had been settled. I think we should clear things out and stop discussing about money.

And thanks for the invitation.

**Comment by mj-xmr** [comment]
@nahuhh : you are 2 steps behind.
Would you like another donut instead?

**Comment by nahuhh** [comment]
@mj-xmr 🤫. 11hrs.

**Comment by mj-xmr** [comment]
I also have to address this now, not to take up too much time during the meeting (thanks for the tip, @nahuhh )

> Work done as per your proposal is work paid. Currently, SoloptXMR:

> Contains no dev tax.

Confirmed 

> With initial setup effort from the end user, external readings via sensors/cameras will control a mining rig.

Yes, and through a level of abstraction, that I created, ANY signal can bound to the input feedback in form of either voltage, Ampere-hours or simply percentage of charge. This is a premeditated form of preparation for Endor's **future** signals as well.

> Now that mj has completed their part of MS3+4 / endor requires no further help / their working relationship is over -
reasons for withholding MS3 (and now MS4) may need to be reevaluated.

It's good to hear, but I'm still here and not abandoning anything.

> During a [Meeting] 3 months ago, it was decided to withold mj's partial payment of milestone 3 (MS3) in the best interests of the community. This would: Prevent project abandonment,  Guarantee that endor had encouragement/help with the uncompleted MS2 + their part of MS3

No. That's not how it works and your predictions about my behavior were wrong. They only ended up getting confirmed for different, only correlated reasons. I finalized the project in an end-2-end manner, because I made my promises to the Community via our social contract. If I didn't care about keeping promises, I'd have abandoned the project right after the denial of payment for my part of M3, especially because using the logic from that meeting, I wouldn't get paid for M4 either, knowing what was brewing with endor's situation. But I don't blame him for that.

@nahuhh 

> He wrote the proposal himself, and in his own words guaranteed he would complete endors parts. Around the time he was threatening people, he said he can wait for pay and always waits for the community. This time he's just being a slimeball in public.

Yes, I wrote the joint proposal myself, as per @Rucknium 's initiative of involving students, and I will never do it again. I discourage anybody from doing anything like that in future as well. This way, exactly through **YOUR actions** Monero will not get new talent involved by experienced devs, but it serves your purpose better: you will definitely enjoy having less competition.

> and in his own words guaranteed he would complete endors parts.

Secondly: this suggests, that your opinion is, that Endor shouldn't get his money then after all, yes? That's one of the reasons why you should rather focus on your donuts more.

> I'm sick and tired of wasting time listening to this guys demands.

I'm sick of you as well.

> We have better things to do than cater to a known scammer.

Looking at your GitHub activity log, you're busier being a poor local politician rather than anything else.

**Comment by mj-xmr** [comment]
BTW. I'm a male, so no `their` please.

**Comment by plowsof** [comment]
Logs 
```
16:00:07 <plowsof> Meeting time https://github.com/monero-project/meta/issues/759

16:00:22 <plowsof> hi

16:00:26 <selenze[m]> escapethe3ra: its important you are here then

16:00:27 <escapethe3ra[m]> hi NotMtth :)

16:00:27 <Stnby[m]> Greetings

16:00:27 <ofrnxmr[m]> Hello

16:00:27 <Siren[m]> Helloo

16:00:28 <NotMtth[m]> escapethe3ra[m]: Xd

16:00:34 <plowsof> luigi1111 (not certain of presence today)

16:00:37 <leito[m]> greetings

16:00:55 <NotMtth[m]> Sup sup

16:01:09 <NotMtth[m]> selenze[m]: It is :)

16:01:10 <plowsof> mj-xmr and endor00 are here i believe

16:01:16 <mj-xmr[m]> Present

16:01:24 <merope> Yep

16:01:36 <gnuteardrops[m]> hi

16:01:40 <NotMtth[m]> Hey gnuteardrops

16:01:46 <plowsof> would you like to say something first mj-xmr / endor?

16:01:50 <ajs_[m]> Hello

16:02:10 <mj-xmr[m]> No. I said everything basically. I'm happy that we could wrap up the review.

16:02:28 <ofrnxmr[m]> Lets get started on news

16:02:33 <merope> Same here

16:02:51 <ofrnxmr[m]> Same here :)

16:03:39 <selenze[m]> halo from little brother here

16:04:06 <plowsof> i have left a statement regarding the situation. i am of the opinion work has been done /
the contributing code of conduct applies to 'extra curricular activities'. and does not effect payment for
hours worked in a ccs. endor will take the work provided by mj and complete the project (without the need of
any help)

16:04:24 <hinto[m]> hello

16:04:28 <monerobull[m]> Hello

16:04:50 <NotMtth[m]> Hello there

16:05:08 <mj-xmr[m]> plowsof: Looks good to me. Especially since this doesn't hinder my payment and will allow
Endor to work more independently.

16:05:18 <mj-xmr[m]> That's a learning experience.

16:06:15 <plowsof> i do however support applying the code of conduct rules (for your behaviour outside of the
ccs) but thats a separate matter

16:06:33 <ofrnxmr[m]> Am I against paying mj? No. Im against paying him before the milestones are completes
as-advertised.

16:06:33 <ofrnxmr[m]> 2 reasons.

16:06:33 <ofrnxmr[m]> 1. He guaranteed, in writing, in the proposal, that he would own project and complete
all parts

16:06:33 <ofrnxmr[m]> 2. He never asked through any proper channels. He just tried to lie cheat and steal

16:07:18 <plowsof> reg time: we do not have to discuss paper wallet / monero shopping / monero archive or bp++

16:07:28 <ofrnxmr[m]> As stated in #2 - its the same issue

16:07:30 <Siren[m]> From the sounds of his proposal mj took a managerial role to ensure completion

16:07:51 <Siren[m]> Or was supposed to*

16:07:52 <monerobull[m]> plowsof: All close?

16:08:12 <MoneroBro> Whats the "Verdict" on M0nero.Shopping?

16:08:14 <plowsof> the guarantee of completion - endor has almost completed ms2 - and has worked on m3/m4 - mj
is not going to get any of endors hours currently.

16:08:14 <ofrnxmr[m]> And in the comments he owns the project and endor is only allowed to participate.

16:08:30 <ofrnxmr[m]> Messy. Not our problem. Pay as-advertised

16:08:52 <Siren[m]> MoneroBro: We'll get to that later

16:09:07 <plowsof> if endor had abandoned the project entirely (he hasn't) then mj must complete

16:09:16 <merope> plowsof: I have not worked on m3/m4 yet, only mj so far. My part is independent though

16:09:23 <mj-xmr[m]> plowsof: It's fine. It was a shock for me at the beginning, but if endor strives to put
in so much work and will hold on to the promise, it's only better for the project and its recipients.

16:09:47 <ofrnxmr[m]> I dont believe the promises of liars and cheats

16:09:54 <monerobull[m]> plowsof: I'm interested in this, would CoC prevent something like funding desnakes
decentralized alphabay? (Not that he needs our money😂)

16:10:01 <ofrnxmr[m]> Who write guarantees and try to merge request their way out of them

16:10:20 <plowsof> endor can indeed complete without any extra help - so mj has completed all of his work

16:10:51 <selenze[m]> mj-xmr[m]: I conquer without wasint time, if he had the managerial authority from the
first, then he should work on it to complete the project or invite some other or even bring forward if there
are any issues between the two where the community may intervene.

16:10:53 <ofrnxmr[m]> Mj's work involves completing the entire project. Otherwise this is not the proposal
that was agreed upon

16:11:35 <plowsof> we've decided mj is not claiming any of endors hours (so he would have to complete the
entire project for free - and he cant  , because endors speciality is the missing pieces)

16:11:48 <Siren[m]> monerobull[m]: Well if it ensures making death threats etc. gets in the way of funding, I
think it would be effective.

16:11:57 <ofrnxmr[m]> Again, I dont remember mj ever requesting, but demanding pay that was not as agreed

16:12:23 <monerobull[m]> Kinda agree with ofrnxmr. We can't have teams apply for ccs, break up, pay out and be
left with a half-baked result

16:12:25 <mj-xmr[m]> I did work my arse out to complete it. Inviting others would have to mean either kicking
endor out which would be really filthy, or abusing the budget which didn't suffice for that either.

16:12:46 <monerobull[m]> But if soloptxmr works as advertised there's no reason not to pay out Mjs work

16:12:46 <plowsof> the ccs is as little red tape as possible - these egregious acts (behaviour etc) need to be
punished outside of the ccs system - which is simply work done , work paid - endor has the framework to
complete the entire project

16:12:57 <nioc> so no endor hours but also milestones not completed

16:13:01 <ofrnxmr[m]> "Kicking endor out" "joint proposal"

16:13:01 <ofrnxmr[m]> Lets vote

16:13:40 <ofrnxmr[m]> I vote no to the exception

16:13:42 <plowsof> milestone 1 - minimum viable product - paid out.

16:13:54 <plowsof> the rest are additions of documentation / readings from sensors / automation - all done

16:14:07 <plowsof> 96 hours * 2 of work done

16:14:11 <monerobull[m]> Kind of out of the loop, what does endor even need to do at this point?

16:14:24 <mj-xmr[m]> There's a plenty of documentation there from me, indeed.

16:14:39 <Siren[m]> monerobull[m]: Make sure it runs on other hardware, not just mj's setup (homeassistant)

16:14:52 <Siren[m]> * Make sure it runs on other hardware, not just mj's setup (aka integrating homeassistant)

16:14:52 <merope> monerobull[m]: mj's part of m3+m4 is done, so he did deliver that. and my work on those
milestones is parallel/independent from that, so there's no reason to hold back his payment for it

16:15:01 <mj-xmr[m]> Siren: For that I used Github's actions.

16:15:01 <plowsof> "with initial setup effort" it works as claimed

16:15:08 <Stnby[m]> Can we continue on this topic later?

16:15:31 <ofrnxmr[m]> Stnby[m]: Lets come back right before the hour is up??

16:16:36 <ofrnxmr[m]> Running in short time lets go

16:16:36 <selenze[m]> I vote this should't happen again, hence not to see it separately but as a whole with
full management and authority of projects on someone

16:17:13 <plowsof> right, ok, will make a decision with luigi

16:17:18 <Siren[m]> I suggest we do not forbid teams working together but perhaps introduce a penalty

16:17:36 <Siren[m]> * forbid teams from working together

16:17:36 <monerobull[m]> is this the first time something like this happaned

16:17:39 <Siren[m]> Seems so

16:17:44 <merope> Siren[m]: why would you ever penalize teams?

16:18:05 <plowsof> every ruling is a precedent so its very important (and will be applied to 'people you like'
one day)

16:18:05 <ofrnxmr[m]> plowsof: No

16:18:07 <monerobull[m]> merope: if they split up?

16:18:11 <plowsof> anyway we've all had our input i think

16:18:13 <Siren[m]> merope: So this type of thing doesn't happen. So people don't take on managerial tasks
when they clearly can't manage.

16:18:21 <Stnby[m]> What penalty? If your team fucks up and shit like this happens, you will have hard time
getting accepted in CCS next time

16:18:21 <ofrnxmr[m]> plowsof: Were coming back to vote at the end ofnthe meeting

16:18:22 <mj-xmr[m]> monerobull[m]: Actually inverse things happened - projects were not finalized even though
there were payouts. Here I had no payout for my M3, but finalized anyway.

16:18:51 <ofrnxmr[m]> For "my" m3

16:18:51 <ofrnxmr[m]> What am imagination you have

16:19:04 <plowsof> right lets move on to the other ideas (skipping paper wallets as they want to close it)

16:19:06 <ofrnxmr[m]> There is one m3, not 2. Mr merge request

16:19:17 <ofrnxmr[m]> plowsof: Please. Running low on time

16:19:20 <plowsof>   b. [Metronero checkout](https://repo.getmonero.org/monero-project/ccs-
proposals/-/merge_requests/353)

16:19:30 <mj-xmr[m]> Stnby[m]: Or the quite the opposite - if you're not treated right, you'll never start a
CCS Proposal again.

16:19:40 <Siren[m]> Good

16:19:44 <plowsof> they have updated their proposal within the last hour (metronero) so needs some thoughts /
opinions

16:19:44 <Siren[m]> Good now metronero

16:19:52 <monerobull[m]> metronero looks decent from what ive heard, are they ready? any new problems since
last meeting=?

16:20:02 <ofrnxmr[m]> Mertronero checkout as I understand is 230xmr total and includes 1 year dog custodial
service + maintenance (post release)for free?

16:20:08 <plowsof> plugins have been requested + maintenance in the last meeting

16:20:18 <Siren[m]> monerobull[m]: We are asking for more XMR now due to the addition of 2 more milestones

16:20:32 <Siren[m]> Siren[m]: These two milestones are for plugins and maintanence

16:20:52 <ofrnxmr[m]> Compared to Previously 100xmr without plugins, maintenance or custodial service

16:20:56 <ofrnxmr[m]> Is this correct?

16:21:04 <Siren[m]> Correct

16:21:06 <monerobull[m]> i like the hosting of feeless custodial version

16:21:25 <ofrnxmr[m]> Same ^ allows for adoption to take place faster

16:21:38 <Stnby[m]> monerobull[m]: Yes last milestone is hosting + code naintenance of  custodial feeless
instance

16:21:39 <monerobull[m]> woocommerce plugin too

16:22:01 <Stnby[m]> Woocomerce and Syllius

16:22:12 <monerobull[m]> would xmr halving in price halt development?

16:22:25 <ofrnxmr[m]> cryptogrampy:

16:22:32 <Siren[m]> monerobull[m]: No

16:23:07 <monerobull[m]> sounds good

16:23:16 <ofrnxmr[m]> Is there a timeline for delivery

16:24:00 <Siren[m]> We'll start as soon as it gets merged and then it should be within the weeks specified in
the milestones? So about 10 weeks

16:24:08 <Siren[m]> And another 5 for plugins

16:24:21 <Siren[m]> Siren[m]: We might be done earlier than 5 weeks

16:24:32 <ofrnxmr[m]> Sounds good

16:24:43 <ofrnxmr[m]> (Was just wondering)

16:24:47 <Stnby[m]> Plugins might require a head start to see the required api functionality

16:25:01 <Stnby[m]> But we are 2 people and can work concurrently

16:25:20 <ofrnxmr[m]> And in the case one if you falls off the earth?

16:26:11 <plowsof> then we use precedent to decide what happens ..

16:26:13 <Stnby[m]> ofrnxmr[m]: I guess the amount stays same but time of delivery is far longer

16:26:21 <selenze[m]> any specific reason why you chose the Affero license.

16:26:29 <ofrnxmr[m]> The other will complete? No secret merge requests for half pay?

16:26:34 <Siren[m]> Nice question ;) honestly whatever the community agrees on. We won't sue or give you 2
weeks to resolve or anything.

16:26:51 <monerobull[m]> plowsof: please no. we want to fund stuff that actually gets finished as proposed

16:26:53 <Stnby[m]> selenze[m]: So no one steals the code and make a proprietary custodial instance

16:26:55 <Siren[m]> selenze[m]: In addition to GPLv3 it offers protection for SaaS

16:27:10 <monerobull[m]> the first case of a problem happening should lead to solid rules for next time

16:28:21 <plowsof> because the changes are so new, its not expected for a decision right now, we need more
eyes on the proposal // advertising / updoots . i am supportive of the proposal and the team

16:28:25 <ofrnxmr[m]> Were about 10 minutes.  licencing to be confirmed by core b4 merge, are we ready to
vote?

16:28:43 <Siren[m]> monerobull[m]: Maybe in that case it would make sense to cancel proposals immediately and
payout, have the members submit separate proposals if they wanna keep working on it.

16:28:50 <plowsof> if yo uwant to voice your vote on it that would be helpful yes

16:29:24 <monerobull[m]> Siren[m]: that leaves us paying for unfinished projects, which we dont want to do

16:29:51 <hinto[m]> should maintenance + xmr volatility be discussed per ccs? i feel like it's common enough
where there should be some common guidelines, or is that too much red tape?

16:30:08 <ofrnxmr[m]> Not volitility.

16:30:32 <monerobull[m]> some people are based and would work no matter how much the price changes. others
should go to MAGIC...

16:30:33 <merope> volatility risk is on the proposer, this has been discussed before

16:30:38 <plowsof> ask / expect the proposer provide deadlines to complete the work 'or else' (or else what)

16:30:38 <ofrnxmr[m]> Your rates should be in xmr and you should accept what you ask for. Or use a platform
that respects fiatm

16:30:51 <Siren[m]> Siren[m]: We saw cases where companies were taking GPLv3 projects such nextcloud and offer
a paid file storage service to their customers. Which technically isn't a GPLv3 violation but that would
certainly violate AGPL

16:31:01 <merope> nobody complains if their xmr were to increase in value, and it would be unreasonable to pay
them less xmr in that case

16:31:11 <Siren[m]> * We saw cases where companies were taking GPLv3 projects such nextcloud and offering a
paid file storage service to their customers. Which technically isn't a GPLv3 violation but that would
certainly violate AGPL

16:31:11 <merope> which means that the same must apply in case of decreases

16:31:20 <ofrnxmr[m]> Wer burning the clock. If were voting I vote move to funding with approval from core on
the license

16:31:34 <monerobull[m]> +1

16:32:17 <ajs_[m]> Regarding licensing, we talked about the issue before and made a PR to have clarify this
issue, but it was never merged and I closed it  https://repo.getmonero.org/monero-project/ccs-
front/-/merge_requests/34/diffs

16:32:21 <ofrnxmr[m]> Anyone voting no or hold?

16:32:25 <monerobull[m]> Total so far: 3 (irc doesnt do reactions, right?)

16:33:15 <mj-xmr[m]> <Siren[m]> "Nice question ;) honestly..." <- OK. I'm very happy that my situation helps
you to shine. Anytime is great time to pump yo rep.

16:33:21 <plowsof> in this case, we just need more eyes on it, and updoots as its new changes

16:33:47 <ofrnxmr[m]> Were all looking at it now.

16:33:51 <selenze[m]> merope: I think projects should only be specified in XMR only ...leaving the economics
to the proposer

16:33:56 <ofrnxmr[m]> And we have a healthy attendance

16:34:24 <selenze[m]> And can someone knowledgeable on all this stuff comment it

16:34:35 <ofrnxmr[m]> The changes were requested changes

16:34:36 <monerobull[m]> theres like 10 people here but only 3 voted

16:34:37 <Stnby[m]> selenze[m]: Obviously it can increase it can decrease, not like in case of XMR value
increase people lower their CCS

16:34:54 <Stnby[m]> * Obviously it can increase it can decrease, not like in case of XMR value increase people
lower their CCS amounts

16:34:55 <plowsof> as a rule of thumb: major milestone changes an hour ago require some time to think about

16:34:56 <merope> selenze[m]: That's exactly how it is right now. People typically include the fiat equivalent
in the calculation to show how they came up with the numbers, but only the final xmr figure matters

16:35:00 <ofrnxmr[m]> Right. So shall people vote no or to wait?

16:35:51 <selenze[m]> merope: I think the calculations made on the page should be eliminated from now on.

16:36:20 <spacekitty420[m]> selenze[m]: like been saying for years, fiat valuation fluctuates but cost of
mining doesnt fluctuates as much so when making one of those proposal it would be better (imo) to think in
term of cost of mining rather than fiat value

16:36:28 <Stnby[m]> selenze[m]: Even monerokon ccs has calculations :)

16:36:32 <plowsof> will spread/advertise the new changes and get more feedback / updoots for metronero.

16:36:35 <selenze[m]> plowsof: my vote is to hold on for sometime until some comments enrich it or we do some
research on this

16:36:48 <plowsof>   d. [CypherPunk Radio](https://repo.getmonero.org/monero-project/ccs-
proposals/-/merge_requests/357)

16:37:03 <NotMtth[m]> TIme has come

16:37:06 <plowsof> last meeting we wanted to see some code or proof of concepts NotMtth

16:37:07 <NotMtth[m]>  * Time has come

16:37:21 <ofrnxmr[m]> Im abstaining from vote on radio.

16:37:21 <monerobull[m]> did they release any interviews already?

16:37:21 <plowsof> do you have anything to share

16:37:43 <leito[m]> did I miss c.  [xmr-btc-swap development and
improvement](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/355)?

16:37:46 <plowsof> oops

16:37:50 <selenze[m]> the content it provides is not satisfying, I thought it would be on to monero mainly

16:38:01 <plowsof> apologies leito

16:38:06 <hinto[m]> selenze[m]: agreed

16:38:22 <NotMtth[m]> Yes, currently something is online, like the I2P domain.

16:38:22 <NotMtth[m]> I still working on the stack, that will be based on a central API.

16:38:23 <NotMtth[m]> Still doing some work and I will update the CCS as I deploy the test servers out

16:38:36 <leito[m]> plowsof: no worries, got some update on it when we go on topic

16:39:03 <plowsof> ok nice, get those changes added onto the ccs proposal so people can see (then people can
make a more informed decision)

16:39:06 <NotMtth[m]> cypherpunk.i2p is the I2P mirror, that is just a raw thing for now

16:39:46 <plowsof> we'll look and provide feedback on your new developments NotMtth

16:40:06 <NotMtth[m]> plowsof: As we discussed I will, it's not complete and I have to do more work on it and
I will update you and community as I make progress on it

16:40:16 <NotMtth[m]> > <@plowsof:matrix.org> ok nice, get those changes added onto the ccs proposal so people
can see (then people can make a more informed decision)

16:40:16 <NotMtth[m]>  * As we discussed I will, it's not complete and I have to do more work on it and I will
update you and community as I make progress

16:40:17 <plowsof> thank you for attending and sharing your progress

16:40:30 <plowsof> appreciated

16:40:36 <plowsof> onto leito   c. [xmr-btc-swap development and
improvement](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/355)

16:40:37 <NotMtth[m]> Thank you for the help plowsof

16:41:02 <selenze[m]> The theme "cypherpunk radio" is well and good but I guess it need to shaped and oriented
to the monero aspirations.

16:41:13 <leito[m]> Okey, big update is: I have decided that I will recluse myself from the rpc api
development, this other guy is doing good work. Obviously I will CR and help him in any questions he has, I
will be changing later today the PR to reflect that

16:41:46 <plowsof> leito you still have translations listed as a milestone / 26 xmr for docker container
(ignoring advice given already) kayaba doesnt like it (see's comit as a money pit) - binarybaron left a
comment too.

16:41:49 <leito[m]> also I will remove the transalation of docs

16:42:02 <plowsof> thanks

16:42:23 <plowsof> binarybaron can complete the gui without the api

16:42:57 <leito[m]> plowsof:  looking back now, I do agree 26 xmr is to much money, what do you thing guys
would be an acceptable amount or should we just bounty it?

16:43:16 <ofrnxmr[m]> Ask after meeting

16:43:25 <ofrnxmr[m]> Lets move on if were not ready

16:44:06 <ofrnxmr[m]> Speed run moneroshoppin, ray, bp+  so we can finish with   Monerokon and solopt

16:44:10 <plowsof>   h. [Draft: MoneroKon 2023 CCS-1](https://repo.getmonero.org/monero-project/ccs-
proposals/-/merge_requests/362)

16:44:11 <cryptogrampy[m]> I left my feedback on the Metronero CCS Siren Stnby .   Looking forward to using it
^_^

16:44:19 <Siren[m]> plowsof: I think writing docker files should be worth less

16:44:29 <Stnby[m]> cryptogrampy[m]: Saw it. 🫀

16:44:49 <ofrnxmr[m]> > <@plowsof:matrix.org>   h. [Draft: MoneroKon 2023
CCS-1](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/362)

16:44:49 <ofrnxmr[m]> >

16:44:49 <ofrnxmr[m]> LAST

16:44:56 <cryptogrampy[m]> Stnby[m]: Will you guys have a docker image for easy deploy?

16:45:02 <Siren[m]> > <@plowsof:matrix.org>   h. [Draft: MoneroKon 2023
CCS-1](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/362)

16:45:02 <Siren[m]> >

16:45:02 <Siren[m]> FUND FUND FUND

16:45:06 <Siren[m]> cryptogrampy[m]: Of course

16:45:10 <ofrnxmr[m]> Speed run through the other stuf asap

16:45:13 <Stnby[m]> cryptogrampy[m]: Yeah

16:45:22 <Stnby[m]> Already have a Dockerfile

16:45:23 <spacekitty420[m]> ofrnxmr[m]: ccs is to fund already properly thought out projects, ray not monero
related, u meant bp++?

16:45:31 <ajs_[m]> Monerokon draft is done for the most part (estimates need to be slightly nudged and commits
squashed) and just awaits final approval from the community

16:45:32 <plowsof> ajs_ has been planning this with the events team, made alot of changes already in response
to feedbacks

16:45:49 <ofrnxmr[m]> Yea bp++ audit ccs

16:46:07 <plowsof> no need to discuss bp++, waiting on the creator to submit new paper

16:46:24 <monerobull[m]> did hinto fall asleep on his keyboard

16:46:34 <ofrnxmr[m]> That needs to be said ^^

16:47:03 <Siren[m]> monerobull[m]: Saturday night :))

16:47:06 <ofrnxmr[m]> Raytina archive > close

16:47:06 <ofrnxmr[m]> Monroshopping > close

16:47:20 <ofrnxmr[m]> Bp++ > hold

16:47:22 <selenze[m]> plowsof: also please do some scheduling on the monero moon, Revuo monero and the Monero
Observer (Blitz)

16:47:33 <ofrnxmr[m]> Moneromoon?

16:47:41 <plowsof>   i. [escapethe3RA Monero Observer maintenance (Winter
2022)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/363)

16:47:44 <ofrnxmr[m]> Hasnt had a release in 6 months

16:48:06 <plowsof> +1 for observer , quality service

16:48:14 <monerobull[m]> +1 for observer, he always writes about the stuff i do /s

16:48:17 <gnuteardrops[m]> +1 Observer

16:48:24 <Siren[m]> +1 Observer

16:48:30 <hinto[m]> monerobull[m]: im holding back writing dumb stuff

16:48:31 <ajs_[m]> Any further questions, comments, concerns, suggestions for monerokon proposal?

16:48:42 <plowsof> only support ajs_

16:48:44 <ofrnxmr[m]> +1 though I have the same requests of observer as I had of moon

16:48:51 <selenze[m]> +1 for Observer except for footnote links and that of revisiting already said
commitments

16:49:20 <plowsof> will leave an updoot on the monerokon

16:49:21 <ofrnxmr[m]> plowsof: Nit from afungible

16:49:28 <merope> +1 observer

16:49:34 <mj-xmr[m]> +1

16:50:16 <plowsof>   f. [MoneroShopping](https://repo.getmonero.org/monero-project/ccs-
proposals/-/merge_requests/360)

16:50:17 <plowsof> close

16:50:25 <monerobull[m]> its fun though

16:50:27 <MoneroBro> Leave open!

16:50:28 <ofrnxmr[m]> Close

16:50:52 <ofrnxmr[m]> Close x5, dont meme vote on your own proposal

16:50:53 <Siren[m]> Close

16:51:03 <selenze[m]> Revuo Monero is bullshit if it was funded

16:51:21 <merope> Close MoneroShopping - they need startup capital, not crowdfunding donations. Cool idea,
wrong place

16:51:22 <MoneroBro> The intend of the Monero.Shopping proposal is to research and document a solution to
increase the "spend-ability" of Monero.

16:51:22 <MoneroBro> The goal is to find a convenient way to pay for any product on the web with Monero.

16:51:23 <MoneroBro> We see this proposal as a "community-think-tank".

16:51:23 <MoneroBro> We don't plan to move to the funding stage until we have a solid blueprint to a clear
goal and the green light from you guys!

16:51:24 <MoneroBro> Keeping this proposal open as a draft would act as a central disscussion and information
gathering board.

16:51:25 <ofrnxmr[m]> Revuo is not funded by ccs. Stay in topic please!

16:51:33 <selenze[m]> even the Monero standard, not expected such a summary for a periodical review

16:51:38 <ofrnxmr[m]> Close. Next

16:51:40 <monerobull[m]> i applaud them for the "buy for me" idea but as ive said, can be done in haveno for
free and requires no trust

16:51:50 <plowsof>   g. [rayatina-archive](https://repo.getmonero.org/monero-project/ccs-
proposals/-/merge_requests/361)

16:51:53 <plowsof> close

16:51:56 <monerobull[m]> close

16:51:57 <ofrnxmr[m]> Close.

16:51:57 <Siren[m]> Close

16:52:15 <ajs_[m]> Close

16:52:40 <merope> Close

16:52:41 <ofrnxmr[m]> Now Monerokon?

16:52:58 <plowsof> here are some monero news outlets (let me know any others if you want them to be listed
selenze ) News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/) -
[The Monero Standard](https://localmonero.co/the-monero-standard)

16:53:28 <plowsof> proposals where discussed ass-backwards here, we've covered them all

16:53:49 <ofrnxmr[m]> ajs_ @ajs_:matrix.org:  are you looking to move to funding at this meeting? Or taking
the draft label off and go to funding after the next?

16:53:54 <selenze[m]> ofrnxmr[m]: can't comment yet

16:53:58 <ofrnxmr[m]> Are you preselling tickets?

16:54:01 <spacekitty420[m]> MoneroBro: like it been said, there's already services doing that (bitrefill
concierge service, proxystore and such), the idea is good, monero funding just doesnt seem to be the place for
it, a reddit post could be more fit to act as discussion thingy / daft and such

16:54:18 <spacekitty420[m]> s/daft/draft/

16:54:54 <plowsof> floor is open, ive seen no convincing argument not to pay mj for his hours

16:54:58 <selenze[m]> spacekitty420: it has already been kindly forwarded to you please do your research

16:55:01 <ofrnxmr[m]> Presales are the best guage of attendance. Even if they are just placeholder tickets at
the higher rate.

16:55:17 <ofrnxmr[m]> plowsof: Your opinion

16:55:20 <Siren[m]> plowsof: Endor agreed with it so

16:55:31 <plowsof> my opinion backed up with statements

16:55:35 <monerobull[m]> just make some rules for next time

16:55:40 <selenze[m]> it has already been kindly forwarded to you please do your research

16:55:50 <mj-xmr[m]> plowsof: Well done in the end, I must admit.

16:55:50 <selenze[m]> that was for Monerobro

16:56:06 <plowsof> managerial role is been negated by endor not abandoning the project

16:56:19 <mj-xmr[m]> monerobull[m]: Note for everybody: don't join milestones. Divide and Conquer.

16:56:33 <ofrnxmr[m]> plowsof: Selsta, myself, siren and others have backed up ours with statements as we''

16:56:54 <ofrnxmr[m]> Well*

16:57:01 <ajs_[m]> ofrnxmr:  after removing draft, it would be ready to be merged

16:57:20 <plowsof> those "punishments" for behaviour  should be taken outside of the ccs , or during the idea
stage

16:57:41 <ofrnxmr[m]> Behavior

16:57:49 <ofrnxmr[m]> When did mj request this exceptio?

16:58:04 <monerobull[m]> in the future we will just have to look more closely at milestones so we don't get
scammed :P

16:58:04 <ajs_[m]> ofrnxmr:  no tickets will be sold once the funding target has been met and location
confirmed

16:58:05 <ofrnxmr[m]> He did a merge request. He didnt request it.

16:58:37 <ofrnxmr[m]> monerobull[m]: Guarantees were made exactly for this reason.

16:59:38 <spacekitty420[m]> s/bitrefill/shopinbit/, s/daft/draft/

16:59:54 <selenze[m]> plowsof: some guidance might be needed for such issues on project mangement and
ownership of deliverance. Should not be repeated I guess

17:00:06 <ofrnxmr[m]> Facepalm

17:00:25 <ofrnxmr[m]> Should not occur, period

17:00:32 <spacekitty420[m]> meeting over or what? everyone can fuck off yet?

17:00:35 <ofrnxmr[m]> Repeated? It hasnt happened yet

17:01:23 <Siren[m]> <plowsof> "managerial role is been negated..." <- I didn't like him taking on a managerial
role and belittling his teammate everywhere (reddit, gitlab)

17:01:25 <merope> Have there been ccs projects with delays/completed past the proposed timeline in the past?

17:01:42 <Siren[m]> Siren[m]: But why are we even discussing this if they both came to an agreement?

17:01:47 <mj-xmr[m]> Siren[m]: I APOLOGIZE. IT WONT HAPPEN AGAIN.

17:01:52 <merope> ofrnxmr[m]: How would you handle delays in work then?

17:01:53 <plowsof> me neither, im agreeing with it all (regarding the behaviour/actions)

17:02:10 <plowsof> but ccs doesnt acknowledge it

17:02:15 <monerobull[m]> merope: pay when its done?

17:02:17 <ofrnxmr[m]> mj-xmr[m]: What part

17:03:00 <ofrnxmr[m]> Dissing endor? Or ____<<<

17:03:14 <ofrnxmr[m]> Threatening to sue ccs

17:03:21 <plowsof> managerial role / finish it all himself isn't an argument anymore (there are no hours being
paid for this)

17:03:34 <ofrnxmr[m]> Demaning pay without a conversation

17:03:34 <ofrnxmr[m]> All of it?

17:03:50 <merope> monerobull[m]: Right. mj's part of those milestones is done, only my independent part is
left - and I'm dealing with some personal issues that are delaying my work, hence the delays

17:04:02 <ofrnxmr[m]> plowsof: Not if you pay him without doing the work.

17:04:09 <merope> Anything else is a separate issue

17:04:12 <plowsof> paying him for the work done though

17:04:21 <mj-xmr[m]> I think endor00 should not be pressured in any way now.

17:04:26 <plowsof> i agree^

17:04:36 <ofrnxmr[m]> mj-xmr:  thats all?

17:04:44 <monerobull[m]> yeah but you have to look at it from the perspective of the ccs, in the future we
dont want to fund things that might not even be functional at all (thank god soloptxmr is)

17:05:16 <merope> monerobull[m]: The fact that it is functional is why we're asking this

17:05:22 <mj-xmr[m]> Ehm.. generally I agree, but this is a special situation.

17:05:36 <plowsof> milestone 1 payout proved it was functional already

17:05:48 <ofrnxmr[m]> So, shall you officially request pay?

17:05:50 <monerobull[m]> should we really pay for someone to develop a frontend that gets abandoned because
his teampartner dropped off? of course not, they applied for a "contract" as a team and the team will deliver
or not get paid

17:05:59 <merope> ofrnxmr[m]: The only work left is my part of those milestones (which will get paid when I
complete them)

17:06:12 <monerobull[m]> * get paid (just an example)

17:06:31 <mj-xmr[m]> endor hasn't abandoned it, decided to do it in his own time, and IF THAT FAILS, I can
always jump in. Deal?

17:06:31 <merope> monerobull[m]: I'm not dropping off. Just shifting my timeline

17:06:41 <ofrnxmr[m]> mj-xmr[m]: Can?

17:06:42 <monerobull[m]> talking about the future

17:06:48 <monerobull[m]> alright im gone, nice meeting

17:06:55 <ofrnxmr[m]> Went from guarantee to "I can"

17:07:37 <ofrnxmr[m]> mj-xmr[m]: It did fail and you tried to bully your way into getting paid and running iff

17:07:47 <ofrnxmr[m]> Only because it didnt work are you here making concessions

17:09:16 <mj-xmr[m]> Yep. Thanks.

17:09:54 <plowsof> leito please share here when you have edited your proposal and we can look during the week
.

17:10:26 <ofrnxmr[m]> So.. are you going to ask?

17:10:31 <plowsof> thank you for all attending , apologies on going over time

17:10:58 <ofrnxmr[m]> MY apologies for going over time

17:11:31 <leito[m]> plowsof: Sure thing 😃

17:11:55 <ofrnxmr[m]> ofrnxmr[m]: I cant possible vote yes on a request that is never made to us.

17:13:52 <ofrnxmr[m]> If you need an exception, you _should_ have to request it. Not demand it and somehow
stumble your way into getting it.

17:13:52 <ofrnxmr[m]> I say _should_ because obviously people say "last time" every time around here.

17:20:34 <plowsof> we can discuss applying the rules here (under project administration) for mj
https://github.com/monero-project/monero/blob/master/docs/CONTRIBUTING.md

17:20:58 <plowsof> ill make a meta issue for daddy Core

17:21:01 <ofrnxmr[m]> Fuck the rules

17:21:17 <ofrnxmr[m]> You dont need a rule for "dont be a slimeball"

17:22:04 <ofrnxmr[m]> ofrnxmr[m]: Half serious.. obviously they might need updating but "rules" dont mean
ANYTHING when you male exeptions because someone talks shit about you online

17:22:39 <ofrnxmr[m]> Wen mj going to ask got an exception?

17:24:25 <ofrnxmr[m]> For*

17:24:25 <ofrnxmr[m]> Or do we just make demands, make noise, and like magic, contracts rewrite themselves

17:27:21 <plowsof> i miss rayatina

17:27:33 <plowsof> always off topic but never mean

17:30:31 <selenze[m]> plowsof: today's list is too many!

17:30:46 <ofrnxmr[m]> plowsof: Im never mean

17:31:43 <plowsof> selenze too many newsletters? they all present info in their own way , people might prefer
one over the other

17:32:33 <selenze[m]> plowsof: I do believe in alternatives...but the others are shits..no work is done onto
them...just a repeatition

17:33:40 <selenze[m]> plowsof: And can you forward escapethe3A to fix the footnote links (some of them do not
work) and that to continue working on the monero announcement, the cypherpunk series

17:34:37 <plowsof> i would like to list all active monero news / newsletters , yes i will ask about the link
issue (is it some personal thing of not directly linking to sites that are not meeting some personal criteria?
i dont know)

17:38:37 <MoneroBro> plowsof  Thank you for inviting me today,  I will sit down with my team and we will
discuss where we go from here. Keep up the good work!

17:41:51 <selenze[m]> plowsof: is there any monero wiki page

17:42:16 <plowsof> MoneroBro has produced a summary of the "current state" of the ecosystem .. a snapshot in
time of this community .... bringing everyone together during the holiday period


```
Automated by [this](https://github.com/plowsof/post-libera-meeting-logs)
